pkg/registry/registry_to_model.go

```diff
@@ -0,0 +1,241 @@
+package registry
+
+import (
+	"encoding/json"
+	"fmt"
+	"strings"
+
+	"github.com/operator-framework/operator-registry/pkg/model"
+	"github.com/operator-framework/operator-registry/pkg/property"
+)
+
+func CreateModelFromBundle(b *Bundle) (*model.Model, error) {
```

Comment:
Move logic from here to `model.Model.AddBundles([]model.Bundle)`?

We need a slice of `model.Bundle` because a `model.Bundle` is channel-specific, and a registry.Bundle can be a member of multiple channels.

Maybe we want `convertRegistryBundleToModelBundles(b *Bundle) ([]model.Bundle, error)`

Reply:
Going towards `convertRegistryBundleToModelBundles(b *Bundle) ([]model.Bundle, error)`